src/main/java/Duke.java

```diff
@@ -43,6 +43,17 @@ public static void unmarkTaskAndPrint(int taskIndex) {
         System.out.println("\t" + LINE);
     }
 
+    public static void printTask(Task task) {
+        System.out.println("\t" + LINE);
+        System.out.println("\t Got it. I've added this task:");
+        System.out.println("\t  " + task);
+        System.out.println("\t Now you have " + counter + " tasks in your list.");
+        System.out.println("\t" + LINE);
+    }
+
+
+
+
     public static void main(String[] args) {
```

Comment:
try to avoid chunks of codes inside main function, can create different methods and make the main function shorter and clearer~